firebase-firestore/src/main/java/com/google/firebase/firestore/local/LocalDocumentsView.java

```diff
@@ -210,4 +215,33 @@ private MaybeDocument getDocument(DocumentKey key, List<MutationBatch> inBatches
 
     return results;
   }
+
+  /**
+   * It is possible that a PatchMutation can make a document match a query, even if the version in
```

Comment:
Ultranit: some class/function names are backtick-escaped, and some aren't. Consider doing backticks for consistency (`PatchMutation`).

Reply:
Please don't backtick in javadoc. There's no markdown support. Use `<code>`/`</code>`.

Reply:
Thanks. Looking at other comments in the file, they use `{@code foo}.`

Reply:
Done.